webapp/src/projects.tsx

```diff
@@ -145,12 +145,14 @@ export class Projects extends data.Component<ISettingsProps, ProjectsState> {
                 case "codeExample": chgCode(scr, false); break;
                 case "tutorial": this.props.parent.startTutorial(scr.url, scr.name); break;
                 default:
-                    const m = /^\/#tutorial:([a-z0A-Z0-9\-\/]+)$/.exec(scr.url);
+                    const m = /^\/#tutorial:([a-z0A-Z0-9\-\/]+)$/.exec(scr.url); // Tutorial
                     if (m) this.props.parent.startTutorial(m[1]);
                     else {
-                        if (scr.youTubeId && !scr.url)
+                        if (scr.youTubeId && !scr.url) // Youtube video
                             window.open('https://youtu.be/' + scr.youTubeId, 'yt');
-                        else if (/^https:\/\//i.test(scr.url))
+                        else if (/^https:\/\//i.test(scr.url)) // External video
```

Comment:
both paths do the same... we should allow local urls (start with / or https urls only)

Reply:
Fixed